lldb/tools/lldb-dap/Protocol/ProtocolRequests.h

```diff
@@ -118,6 +118,81 @@ bool fromJSON(const llvm::json::Value &, InitializeRequestArguments &,
 /// Response to `initialize` request. The capabilities of this debug adapter.
 using InitializeResponseBody = std::optional<Capabilities>;
 
+/// DAP Launch and Attach common configurations.
+struct DAPConfiguration {
```

Comment:
Nit: Any reason to not call this `Configuration`? We don't prefix everything else here with `DAP` and it's already in the `dap::protocol` namespace. 
```suggestion
struct Configuration {
```

Reply:
Done.